semeio/jobs/design2params/design2params.py

```diff
@@ -5,7 +5,6 @@
 
 import numpy as np
 import pandas as pd
-from xlrd.biffh import XLRDError
```

Comment:
It should be deleted in `setup.py` as well, right?

Reply:
There is also a comment about `xlrd` here:
https://github.com/equinor/semeio/blob/24bc00bb1755ab420bc4b8e1f1be57c7f4096086/tests/jobs/pyscal/test_pyscal.py#L148

but I am not sure I understand what is meant here. Should it be deleted as well?

Reply:
No, we still need xlrd in order to support old Excel file formats. But with newer Pandas versions we do not need workarounds in semeio in order to support it. If we remove xlrd, the `test_support_multiple_format` for design2params will fail.

The pyscal test will also fail if it is missing, but pyscal declares xlrd as its own dependency.

Reply:
So we should declare `xlrd` as a direct dependency of `semeio` even though we only use it indirectly through Pandas, but Pandas declares it as optional.